### ISSUE TITLE
`this.homey.flow.getToken` is not a function

### ISSUE THREAD

**Issue by robertklep** [athombv/apps-sdk-issues#277]
[This function](https://apps-sdk-v3.developer.homey.app/ManagerFlow.html#getToken) doesn't seem to be implemented in the SDK:
```
TypeError: this.homey.flow.getToken is not a function
```

**Comment by WeeJeWel** [comment]
Can you share a code snippet? The method does exist in the SDK.

**Comment by robertklep** [comment]
```
const Homey = require('homey');

module.exports = class MyApp extends Homey.App {
  async onInit() {
    this.homey.flow.getToken('some-token');
  }
}
```

**Comment by WeeJeWel** [comment]
Is this SDK v3?

**Comment by robertklep** [comment]
Yes, other functions, like `createToken()`, do work.

homeyVersion "8.1.0"

https://community.homey.app/t/gettoken-for-flows-not-working/73834/

Also nowhere to be found in the TS typings:
```
declare class ManagerFlow extends Manager {
    static ID: string;
    /**
     * @param {string} id The ID of the card as defined in the app's `app.json`.
     * @returns {FlowCardAction}
     */
    getActionCard(id: string): FlowCardAction;
    /**
     * @param {string} id The ID of the card as defined in the app's `app.json`.
     * @returns {FlowCardCondition}
     */
    getConditionCard(id: string): FlowCardCondition;
    /**
     * @param {string} id The ID of the card as defined in the app's `app.json`.
     * @returns {FlowCardTrigger}
     */
    getTriggerCard(id: string): FlowCardTrigger;
    /**
     * @param {string} id The ID of the card as defined in the app's `app.json`.
     * @returns {FlowCardTriggerDevice}
     */
    getDeviceTriggerCard(id: string): FlowCardTriggerDevice;
    /**
     * The FlowToken class can be used to create a Tag in the Flow Editor.
     * @param {string} id - ID of the token, should be alphanumeric.
     * @param {object} opts
     * @param {string} opts.type - Type of the token, can be either `string`, `number`, `boolean` or `image`.
     * @param {string} opts.title - Title of the token
     * @returns {Promise<FlowToken>}
     */
    createToken(id: string, opts: {
        type: string;
        title: string;
    }): Promise<FlowToken>;
    /**
     * Unregister a {@link FlowToken}.
     * @param {FlowToken} tokenInstance
     * @returns {Promise<any>}
     */
    unregisterToken(tokenInstance: FlowToken): Promise<any>;
}
```

**Comment by WeeJeWel** [comment]
Hm, I cannot seem to reproduce this...

```js
    const myStringToken = await this.homey.flow.createToken('my_string_token', {
      type: 'string',
      title: 'My String Token',
    });
    const myStringToken2 = this.homey.flow.getToken('my_string_token');
    this.log('myStringToken2', myStringToken2)
```

```
[log] 2022-12-19 13:32:32 [TestSuiteApp] myStringToken2 <ref *2> FlowToken {
  manager: <ref *1> ManagerFlow {
    _events: [Object: null prototype] {
      __crash: [Function: bound _onManagerCrash],
      __debug: [Function (anonymous)],
      __log: [Function (anonymous)],
      __error: [Function (anonymous)]
    },
   ...
```

**Comment by WeeJeWel** [comment]
Also in the thread:

```js
async onInit() {
    this.homey.flow.createToken('tomorrowsBestPrice', { type: 'string', title: 'Best export price', value: 0 });
    const token = this.homey.flow.getToken('tomorrowsBestPrice');
}
```

Doesn't await `createToken`. The issue also doesn't describe _what_ fails.

**Comment by robertklep** [comment]
Yes, it does:
```
“Unable to initialize app TypeError: this.homey.flow.getToken is not a function”
```

My snippet produces the same error. Your example code does too.

**Comment by WeeJeWel** [comment]
Can you share the entire source of your test case?

**Comment by robertklep** [comment]
https://github.com/robertklep/get-token-test

**Comment by WeeJeWel** [comment]
Thank you. I see why I couldn't reproduce it now: this has been fixed in v8.1.0 → v8.1.1.

**Comment by robertklep** [comment]
Thanks, I guess I must have missed that in the changelog.
